html/webappapis/scripting/reporterror.any.js

```diff
@@ -0,0 +1,22 @@
+setup({ allow_uncaught_exception:true });
+
+test(t => {
+  let happened = false;
+  self.addEventListener("error", t.step_func(e => {
+    assert_equals(e.error, 1);
```

Comment:
I think it'd be valuable to test `filename` equals `location.href`, and test that `lineno` and `colno` are > 0. Otherwise an implementation could get lazy with reportError().